pkg/actors/actors.go

```diff
@@ -287,9 +287,9 @@ func (a *actorsRuntime) startAppHealthCheck(ctx context.Context, opts ...health.
 	for {
 		select {
 		case <-ctx.Done():
-			break
+			return
```

Comment:
Why is this needed?

Reply:
The logic is currently bugged. The `break` statement will break from the `select` block and not the `for` loop.